### PR TITLE
[luci/IR] Add BidirectionalSequenceLSTM op

### PR DESCRIPTION
**Pull request by dr-venkman**
This commit add BidirectionalSequenceLSTM operation on luci/IR.

ONE-DCO-1.0-Signed-off-by: venkat.iyer venkat.iyer@samsung.com

### CODE REVIEW

File: compiler/luci/lang/include/luci/IR/CircleNodes.h [hunk 1]
```diff
@@ -135,6 +136,8 @@
 // Virtual nodes
 #include "Nodes/CircleInput.h"
 #include "Nodes/CircleOutput.h"
+#include "Nodes/CircleBidirectionalSequenceLSTMOut.h"
+
```
Comment (mhs4670go) [inline compiler/luci/lang/include/luci/IR/CircleNodes.h:140]: nit. unintended whitespace?

File: compiler/luci/lang/src/Nodes/CircleBidrectionalSequenceLSTM.test.cpp [hunk 2]
```diff
@@ -0,0 +1,130 @@
+/*
+ * Copyright (c) 2020 Samsung Electronics Co., Ltd. All Rights Reserved
```
Comment (mhs4670go) [inline compiler/luci/lang/src/Nodes/CircleBidrectionalSequenceLSTM.test.cpp:2]:
```suggestion
 * Copyright (c) 2021 Samsung Electronics Co., Ltd. All Rights Reserved
```

File: compiler/luci/lang/include/luci/IR/CircleNodes.h (hunk 1 above)
Comment (dr-venkman) [reply compiler/luci/lang/include/luci/IR/CircleNodes.h]: Corrected, thanks for the comment.

File: compiler/luci/lang/src/Nodes/CircleBidrectionalSequenceLSTM.test.cpp (hunk 2 above)
Comment (dr-venkman) [reply compiler/luci/lang/src/Nodes/CircleBidrectionalSequenceLSTM.test.cpp]: Corrected, thanks for the comment.

**Review by mhs4670go — APPROVED** [review]
LGTM

**Review by struss — APPROVED** [review]
LGTM

**Pull request merged**